x/group/typesupport.go

```diff
@@ -0,0 +1,40 @@
+package group
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/x/group/errors"
+)
+
+func (ms Members) ValidateBasic() error {
```

Comment:
I'm not sure about the difference between `types.go` and `typesupport.go`. For example, can this method go to `types.go`?

Reply:
in `types.go` we usually implement methods for generated types whereas the types defined in `typesupport.go` are only here to extend these types with some util functionalities 
does that make sense?
I don't have a strong opinion whether we put them in `types.go` or `typesupport.go`